trace-telemetry-exporter/src/main.rs

```diff
@@ -0,0 +1,184 @@
+use anyhow::Result;
+use chrono::{DateTime, Utc};
+use clap::Parser;
+use metrics::{counter, gauge};
+use metrics_exporter_prometheus::PrometheusBuilder;
+use rdkafka::{
+    consumer::{stream_consumer::StreamConsumer, CommitMode, Consumer},
+    message::Message,
+};
+use std::net::SocketAddr;
+use supermusr_streaming_types::dat1_digitizer_analog_trace_v1_generated::{
+    digitizer_analog_trace_message_buffer_has_identifier, root_as_digitizer_analog_trace_message,
+};
+
+const METRIC_MSG_COUNT: &str = "digitiser_message_received_count";
+const METRIC_LAST_MSG_TIMESTAMP: &str = "digitiser_last_message_timestamp";
+const METRIC_LAST_MSG_FRAME_NUMBER: &str = "digitiser_last_message_frame_number";
+const METRIC_CHANNEL_COUNT: &str = "digitiser_channel_count";
+const METRIC_SAMPLE_COUNT: &str = "digitiser_sample_count";
+
+#[derive(Debug, Parser)]
+#[clap(author, version, about)]
+struct Cli {
+    #[clap(long)]
+    broker: String,
+
+    #[clap(long)]
+    username: Option<String>,
+
+    #[clap(long)]
+    password: Option<String>,
+
+    #[clap(long = "group")]
+    consumer_group: String,
+
+    #[clap(long)]
+    trace_topic: String,
+
+    #[clap(long, default_value = "127.0.0.1:9090")]
+    metrics_address: SocketAddr,
```

Comment:
Allow this to also be set by environment variable (similar to `OBSERVABILITY_ADDRESS` elsewhere).

Reply:
Yes, that is expected.

Reply:
I'm a bit confused. I thought that `OBSERVABILITY_ADDRESS` could be set by environment variable?

Reply:
It can, but only when configured to do so. Just checking and this does seem to have been omitted elsewhere, could you add the correct flag for these (see https://github.com/DanNixon/satori/blob/main/event-processor/src/main.rs#L30).